src/SweetAlert/SweetAlertNotifier.php

```diff
@@ -223,4 +223,20 @@ private function hasTitle()
     {
         return (bool) strlen($this->config['title']);
     }
+    
+    /**
+     * Make Message HTML view.
+     * 
+     * @param bool|true $html
+     *
+     * @return \UxWeb\SweetAlert\SweetAlertNotifier $this
+     */
+    public function html($html = true)
```

Comment:
ping @uxweb

Thank you i have removed unnecessary $html tag and i don't see it's important .

sorry look like i lost between the requests. 

Thank you.